packages/relayer/src/interfaces/env.ts

```diff
@@ -6,8 +6,9 @@ export interface Env {
   // Data for relayer account
   PRIVATE_KEY: string;
   ACCOUNT_ID: string;
-  // Data for HYC contracts
-  HYC_CONTRACT: string;
   // Fee charged for relaying (0.25 == 25%)
   RELAYER_FEE: string;
+  RELAYER_URL: string;
+  HYC_CONTRACT: string;
+  BASE_STORAGE_FEE: string;
```

Comment:
How should this fee be specified? I'd add a comment and docs.
Should it be in %? Should it be an absolute value?